app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DatasetsManagement.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.HumanLearning2021.HumanLearningApp.model
+
+import com.github.HumanLearning2021.HumanLearningApp.firestore.FirestoreDatasetInterface
+
+object DatasetsManagement: DatasetsManagementInterface {
```

Comment:
This might not be an object. Worst case scenario I'll change it though since this fact is only relevant at dependency injection time.